spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
     describe '.new' do
-      subject(:person) { BlockScore::Person.new(attributes_for(:person)) }
+      let(:attrs)      { attributes_for(:person) }
+      subject(:person) { BlockScore::Person.new(attrs) }
```

Comment:
I mainly provide this suggestion because it offers a consistent guide that can be followed. I have no problem if subject comes first as long as it's consistent throughout the code. I do personally find having `subject` first makes it much easier to find when looking for the object under test -- I view `let` blocks as kind of secondary; they are always subordinate to the subject.